app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java

```diff
@@ -344,25 +348,26 @@ protected void updateCardViewImage(@Nullable String url, @Nullable String blurHa
             try {
                 if (url == null) {
                     Glide.with(mCardView.getContext())
-                            .load(mDefaultCardImage)
-                            .diskCacheStrategy(DiskCacheStrategy.RESOURCE)
-                            .into(mCardView.getMainImageView());
+                        .load(mDefaultCardImage)
+                        .diskCacheStrategy(DiskCacheStrategy.RESOURCE)
+                        .into(mCardView.getMainImageView());
                 } else {
-                    GlideUtils.blurHashPlaceholder(
+                    BlurHashUtils.createBlurHashDrawable(
```

Comment:
I created a new function that accepts the lifecycleowner, blurhash, size and a return function. No context or glide object needed now